src/main/java/com/jalasoft/webservice/controller/ImgController.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ *
+ *  This software is the confidential and proprietary information of Jalasoft.
+ *  ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+
+package com.jalasoft.webservice.controller;
+
+import com.jalasoft.webservice.entitities.*;
```

Comment:
remove not used imports: 
import com.jalasoft.webservice.utils.PropertiesReader;
import org.springframework.http.MediaType;
import org.springframework.http.ResponseEntity;
import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
import javax.servlet.annotation.MultipartConfig;
import java.io.FileNotFoundException;
import java.net.URI;